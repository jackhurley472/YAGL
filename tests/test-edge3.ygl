
Graph G;

Node n("Adam");
Node n2("James");
Node n3("Tank");
Node n4("Jack");
Node n5("Jack2");
Node n6("Jack3");

G + n;
G + n2;
G + n3;
G + n4;
G + n5;
G + n6;

int b = 3;

G: n -> n2 ->2 n3 ->b n4 ->|b + 1| n5 ->|5| n6;

printGraph(G);


