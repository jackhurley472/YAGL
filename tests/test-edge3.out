============== Graph Print ===============
	Stats: 	8	6	5

Nodes:
Node (0) : Adam --- Node (1) : James --- Node (2) : Tank
Node (3) : Jack --- Node (4) : Jack2 --- Node (5) : Jack3


Edges:
[(0) : Adam] ---(1)---> [(1) : James]
[(1) : James] ---(2)---> [(2) : Tank]
[(2) : Tank] ---(3)---> [(3) : Jack]
[(3) : Jack] ---(4)---> [(4) : Jack2]
[(4) : Jack2] ---(5)---> [(5) : Jack3]
============ End Graph Print =============
